build_ng2.xml

```diff
@@ -125,7 +125,13 @@
   
   <target name="bootstrapBuild"><!--We could pass different args for bootstrap if we wanted-->
     <antcall target="npmInstall" unless:set="noInstall">  
-      <param name="packagejson.Location" value="${capstone}/zlux-server-framework/js"/>
+      <param name="packagejson.Location" value="${capstone}/zlux-server-framework"/>
+    </antcall>
+    <antcall target="npmInstall" unless:set="noInstall">  
+      <param name="packagejson.Location" value="${capstone}/zss-auth"/>
```

Comment:
This is very fragile, confusing and difficult to test due to the long build time, in other words, it's a pain. I was very happy that I've done some manipulations and it didn't break in mysterious ways. I'd rather stay away from this build system as much as possible, to avoid bad surprises in the future: who knows how it may break.

Reply:
Are you saying that if I simply remove lines 130-132, it should work?

Reply:
Looks like it does work the same!

Reply:
Incidentally it does, because bluebird is satisfied by the server, and there is no typescript to compile. We can revisit as we move things to typescript and see what build changes that requires.